### PR TITLE
updates and cleans up package.json scripts.

### PR DESCRIPTION
**Pull request by enigmalea**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,31 +2,29 @@
   "name": "bobaserver",
   "version": "1.0.0",
   "description": "Server for BobaBoard",
-  "homepage": "https://docs.bobaboard.com/",
+  "homepage": "https://github.com/essential-randomness/bobaserver#readme",
```
Comment (essential-randomness) [inline package.json:5]: revert these changes?

File: package.json [hunk 2]
```diff
@@ -2,31 +2,29 @@
...
   },
   "license": "MIT",
-  "author": "BobaBoard Team",
+  "author": "",
   "main": "dist/server/index.js",
   "scripts": {
-    "build": "tsc && tsc-alias && copyfiles .env .env.prod firebase-sdk.json server/**/*.sql ./dist",
+    "db:start": "docker compose down && docker compose up",
     "deploy": "gcloud config set project bobaboard && gcloud app deploy --version live",
-    "dev:watch": "cross-env DEBUG=bobaserver:*,-*info DEBUG_DEPTH=10 nodemon --watch . --exec node -r tsconfig-paths/register -r ts-node/register server/index.ts -e yaml,ts,sql",
```
Comment (essential-randomness) [inline package.json:19]: we should add `yarn run dev` that starts both the server and the db. Double-check, but this should work as a script:

`yarn run db:start & yarn run server:dev`

File: package.json [hunk 3]
```diff
@@ -2,31 +2,29 @@
...
     "open-api:docker": "docker compose -f docker-compose.open-api.test.yml run --rm test",
     "open-api:generate": "node -r dotenv/config -r ts-node/register -r tsconfig-paths/register utils/generate-spec-file.ts dotenv_config_path=./.env.prod",
     "open-api:typescript": "yarn run open-api:generate && openapi-zod-client ./open-api-spec.json -o ./types/open-api/generated/schemas.ts --export-schemas --template ./types/open-api/generated/schemas.hbs --with-alias &&  openapi-zod-client ./open-api-spec.json -o ./types/open-api/generated/types.ts --export-schemas --template ./types/open-api/generated/types.hbs --with-alias",
     "open-api:validate": "yarn run open-api:generate && validate-api open-api-spec.json && openapi-examples-validator open-api-spec.json --ignore-formats color quill-delta && openapi lint open-api-spec.json --skip-rule=no-ambiguous-paths",
-    "stage": "gcloud config set project bobaboard && gcloud app deploy --quiet --version staging --no-promote",
-    "start": "cross-env DEBUG=bobaserver:*,-*info node -r dotenv/config dist/server/index.js dotenv_config_path=./.env.prod",
-    "start-db": "docker compose down && docker compose up",
```
Comment (essential-randomness) [inline package.json:26]: Should we consider changing start-db to be like `node -e \"console.error('This command has been deprecated. Use db:start now.')\"`, at least for now? Same for `dev:watch`.

File: package.json [hunk 4]
```diff
@@ -2,31 +2,29 @@
...
     "open-api:docker": "docker compose -f docker-compose.open-api.test.yml run --rm test",
     "open-api:generate": "node -r dotenv/config -r ts-node/register -r tsconfig-paths/register utils/generate-spec-file.ts dotenv_config_path=./.env.prod",
     "open-api:typescript": "yarn run open-api:generate && openapi-zod-client ./open-api-spec.json -o ./types/open-api/generated/schemas.ts --export-schemas --template ./types/open-api/generated/schemas.hbs --with-alias &&  openapi-zod-client ./open-api-spec.json -o ./types/open-api/generated/types.ts --export-schemas --template ./types/open-api/generated/types.hbs --with-alias",
     "open-api:validate": "yarn run open-api:generate && validate-api open-api-spec.json && openapi-examples-validator open-api-spec.json --ignore-formats color quill-delta && openapi lint open-api-spec.json --skip-rule=no-ambiguous-paths",
-    "stage": "gcloud config set project bobaboard && gcloud app deploy --quiet --version staging --no-promote",
-    "start": "cross-env DEBUG=bobaserver:*,-*info node -r dotenv/config dist/server/index.js dotenv_config_path=./.env.prod",
```
Comment (essential-randomness) [inline package.json:25]: I'd do `start` and `build` as synonym of `server:start` and `server:build` as those are sometimes used by automated tools.

**Review by essential-randomness — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (enigmalea) [reply package.json:5]: I'm so confused as to how this happened? I pulled from main before I started working?

**Review by essential-randomness — APPROVED** [review]
